org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/services/format/XMLFormatterTest.java

```diff
@@ -1964,13 +1983,37 @@ public void testAttributeNameValueMultipleLines() throws BadLocationException {
 				"  </a>\r\n" + //
 				"</xml>";
 		String expected = "<xml>\r\n" + //
-				"  <a a=\"aa\">\r\n" + //
+				"  <a\r\n" + //
+				"    a=\"aa\"\r\n" + //
+				"  >\r\n" + //
 				"    <b></b>\r\n" + //
 				"  </a>\r\n" + //
 				"</xml>";
 		assertFormat(content, expected);

```

Comment:
Declare expected te()

Reply:
We need to do this for the old formatter as well?

Reply:
Oh sorrý I didnt noticed that. Please ignore my comment.